.gitattributes

```diff
@@ -11,5 +11,5 @@
 *.txt text
 *.yml text
 
-# Use Rust syntax highlighter for Move and Move IR code
+# Use Move syntax highlighter for Move IR code
```

Comment:
Do we need to do something for `*.move` files? or is that covered by default?

Reply:
> Do we need to do something for `*.move` files? or is that covered by default?

*.move files will use github  default syntax highlighting for Move